csharp/src/Google.Protobuf/project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
```

Comment:
Not sure what you mean by the "old non-dotnet solution". We're still building with the csproj (as far as I can tell - it certainly still exists; VS hasn't deleted it).
No idea whether xbuild supports this though - it doesn't seem to have the relevant targets file.